genie-core/src/main/java/com/netflix/genie/core/jobs/workflow/impl/JobKickoffTask.java

```diff
@@ -82,6 +86,8 @@ public JobKickoffTask(
         this.executor = executor;
         this.hostname = hostname;
         this.timer = registry.timer("genie.jobs.tasks.jobKickoffTask.timer");
+        retryTemplate = new RetryTemplate();
+        retryTemplate.setBackOffPolicy(new ExponentialBackOffPolicy());
```

Comment:
Don't we already have a retry template as a bean somewhere? why isn't this passed into the constructor?